src/crimson/net/SocketConnection.cc

```diff
@@ -663,6 +671,7 @@ seastar::future<> SocketConnection::handle_connect_reply(msgr_tag_t tag)
     return fault();
   case CEPH_MSGR_TAG_BADAUTHORIZER:
     if (h.got_bad_auth) {
+      logger().error("{} got bad authorizer", __func__);
```

Comment:
Is it a temporary solution here to use plain `logger()` instead of `dout` facility?

Reply:
please refer to the commit message at https://github.com/ceph/ceph/pull/23957/commits/f59cb91264bca91a840c1aa84dba4c6165cbceaf .

so, yes, it is an intermediate solution. but we will not likely throw it away once we have the LTTng logging backend. because its interface resembles the 2nd interface i imagined in https://github.com/ceph/ceph/pull/22458#discussion_r206902863 . so we can adapt it to the new interface with minimal efforts.